src/theme/components.tsx

```diff
@@ -212,6 +196,50 @@ export function ExternalLinkIcon({
   )
 }
 
+const ToolTipWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: absolute;
+  transform: translate(5px, 32px);
+  z-index: 9999;
+`
+
+const CopiedTooltip = styled.div`
+  background-color: ${({ theme }) => theme.black};
+  text-align: center;
+  justify-content: center;
+  width: 60px;
+  height: 32px;
+  line-height: 32px;
+  border-radius: 8px;
+
+  color: ${({ theme }) => theme.white};
+  font-size: 12px;
+`
+
+function ToolTip() {
+  return (
+    <ToolTipWrapper>
+      <TooltipTriangle />
```

Comment:
nit: This is overkill, and it should instead be done using a pseudo-element in CSS in the wrapper.

Reply:
good point. @kayleegeorge can you update

Reply:
what specifically do you mean by this @zzmp ?

Reply:
this can be done in pure css without bringing in an svg dependency